### PR TITLE
support pg create table sql for multi version -2

### PR DESCRIPTION
**Pull request by tuichenchuxin**
For #17137.

Changes proposed in this pull request:
- add template for multi version for create table sql

### CODE REVIEW

File: shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-dialect/shardingsphere-data-pipeline-postgresql/src/main/java/org/apache/shardingsphere/data/pipeline/postgresql/ddlgenerator/PostgresConstraintsLoader.java [hunk 1]
```diff
@@ -85,14 +85,20 @@ private void fetchConstraintsColumns(final Collection<Map<String, Object>> const
...
-            Map<String, Object> parameters = new LinkedHashMap<>();
-            parameters.put("cid", each.get("oid"));
-            Collection<Object> includes = new LinkedList<>();
-            for (Map<String, Object> include : executeByTemplate(parameters, "index_constraint/%s/get_constraint_include.ftl")) {
-                includes.add(include.get("colname"));
+            getConstraintInclude(each);
+        }
+    }
+    
+    private void getConstraintInclude(final Map<String, Object> constraintsProp) {
```
Comment (strongduanmu) [inline shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-dialect/shardingsphere-data-pipeline-postgresql/src/main/java/org/apache/shardingsphere/data/pipeline/postgresql/ddlgenerator/PostgresConstraintsLoader.java:92]: @tuichenchuxin Can we rename this method to appendConstraints?

File: shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-dialect/shardingsphere-data-pipeline-postgresql/src/main/java/org/apache/shardingsphere/data/pipeline/postgresql/ddlgenerator/PostgresConstraintsLoader.java [hunk 2]
```diff
@@ -155,8 +161,10 @@ private void getExclusionConstraintsColumns(final Map<String, Object> exclusionC
         Map<String, Object> map = new HashMap<>();
         map.put("cid", exclusionConstraintsProps.get("oid"));
         Collection<String> include = new LinkedList<>();
-        for (Map<String, Object> each : executeByTemplate(map, "exclusion_constraint/%s/get_constraint_include.ftl")) {
-            include.add(each.get("colname").toString());
+        if (getMajorVersion() >= 11) {
```
Comment (strongduanmu) [inline shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-dialect/shardingsphere-data-pipeline-postgresql/src/main/java/org/apache/shardingsphere/data/pipeline/postgresql/ddlgenerator/PostgresConstraintsLoader.java:164]: Can we extract 11 to constant?

**Review by strongduanmu — CHANGES_REQUESTED** [review]

File: shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-dialect/shardingsphere-data-pipeline-postgresql/src/main/java/org/apache/shardingsphere/data/pipeline/postgresql/ddlgenerator/PostgresConstraintsLoader.java (hunk 1 above)
Comment (tuichenchuxin) [reply shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-dialect/shardingsphere-data-pipeline-postgresql/src/main/java/org/apache/shardingsphere/data/pipeline/postgresql/ddlgenerator/PostgresConstraintsLoader.java]: Thank you. Done.

File: shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-dialect/shardingsphere-data-pipeline-postgresql/src/main/java/org/apache/shardingsphere/data/pipeline/postgresql/ddlgenerator/PostgresConstraintsLoader.java (hunk 2 above)
Comment (tuichenchuxin) [reply shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-dialect/shardingsphere-data-pipeline-postgresql/src/main/java/org/apache/shardingsphere/data/pipeline/postgresql/ddlgenerator/PostgresConstraintsLoader.java]: Thank you. Done.

**Review by strongduanmu — APPROVED** [review]

**Pull request merged**